pkg/webhook/fleetresourcehandler/fleetresourcehandler_webhook.go

```diff
@@ -67,14 +67,14 @@ func (v *fleetResourceValidator) Handle(ctx context.Context, req admission.Reque
 		case req.Kind == validation.NamespaceGVK:
 			klog.V(2).InfoS("handling namespace resource", "name", req.Name, "operation", req.Operation, "subResource", req.SubResource)
 			response = v.handleNamespace(req)
-		case req.Kind == validation.V1Alpha1IMCGVK || req.Kind == validation.V1Alpha1WorkGVK || req.Kind == validation.IMCGVK || req.Kind == validation.WorkGVK || req.Kind == validation.EndpointSliceExportGVK || req.Kind == validation.EndpointSliceImportGVK || req.Kind == validation.InternalServiceExportGVK || req.Kind == validation.InternalServiceImportGVK:
-			klog.V(2).InfoS("handling fleet owned namespaced resource in system namespace", "GVK", req.RequestKind, "namespacedName", namespacedName, "operation", req.Operation, "subResource", req.SubResource)
-			response = v.handleFleetMemberNamespacedResource(ctx, req)
+		case req.Kind == validation.V1Alpha1IMCGVK || req.Kind == validation.V1Alpha1WorkGVK || req.Kind == validation.IMCGVK || req.Kind == validation.WorkGVK || req.Kind == validation.EndpointSliceExportGVK || req.Kind == validation.EndpointSliceImportGVK || req.Kind == validation.InternalServiceExportGVK || req.Kind == validation.InternalServiceImportGVK || req.Kind == validation.EndpointSliceGVK:
```

Comment:
Hi Arvind! Is there any particular reason why we are checking for EndpointSlice objects? They are never exported in their original forms by our agents IIRC

Reply:
And if they are created by the user, they should be treated as normal objects like other resources.

Reply:
My bad misread this line and thought the endpoint slice was being created on the hub https://github.com/Azure/fleet-networking/blob/main/pkg/controllers/member/endpointsliceimport/controller.go#L223